kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -359,7 +360,8 @@ private void startGCSToBQLoadTask() {
       BucketInfo bucketInfo = BucketInfo.of(bucketName);
       bucket = gcs.create(bucketInfo);
     }
-    GCSToBQLoadRunnable loadRunnable = new GCSToBQLoadRunnable(getBigQuery(), bucket);
+    Storage.BlobListOption folderPrefixOption = Storage.BlobListOption.prefix(folderName);
+    GCSToBQLoadRunnable loadRunnable = new GCSToBQLoadRunnable(getBigQuery(), bucket, folderPrefixOption);
```

Comment:
does this work as expected if there is no folder prefix configured?

Reply:
I believe so. As the `GCS_FOLDER_NAME_DEFAULT` on the config is set to an empty string, we are filtering for objects with a path prefixed by an empty string which is the same as no filter at all. 

At least that was how I thought it would work, and took the passing integration tests as confirmation :)